### PR TITLE
Add rwd for history

### PR DESCRIPTION
**Pull request by MateuszWasik**

### CODE REVIEW

File: src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss [hunk 1]
```diff
@@ -26,7 +27,37 @@
...
-    margin-top: 30px
+    margin-top: 15px
+  }
+}
+
+@media screen and (max-width: 414px) {
+  .periodButtons {
+    button {
+      height: 30px;
+      font-size: 11px;
```
Comment (skrzepij) [inline src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss]: please use rem unit thanks a lot bro bro brooooo

**Review by skrzepij — CHANGES_REQUESTED** [review]

File: src/components/ChartComponent/ChartComponent.scss [hunk 2]
```diff
@@ -2,3 +2,19 @@
...
+
+@media screen and (min-width: 415px) {
+  .line-chart {
+    height: 600px;
+  }
+}
+
+@media screen and (max-width: 414px) {
+  .chart-container {
+    height: 430px
```
Comment (skrzepij) [inline src/components/ChartComponent/ChartComponent.scss:14]: I think this one is overdone. try to remove it ;) 

`.line-chart` as a child of this container, should expand `.chart-container` height automatically :)

File: src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss (hunk 1 above)
Comment (MateuszWasik) [reply src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss]: Done

File: src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss [hunk 3]
```diff
@@ -26,7 +27,37 @@
...
+
+  .backToDashboard {
+    margin-top: 0;
+  }
+}
+
+@media screen and (max-width: 365px) {
+  .periodButtons {
+    button {
+      height: 30px;
```
Comment (skrzepij) [inline src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss:50]: if this style is set in media query above, then shouldn't be needed here :P

**Review by skrzepij — APPROVED** [review]

**Pull request merged**